web/backend/src/controllers/dela.ts

```diff
@@ -174,6 +176,32 @@ delaRouter.use('/services/shuffle/:formID', (req, res, next) => {
   }
   next();
 });
+
+delaRouter.post('/forms/:formID/vote', (req, res) => {
+  if (!req.session.userId) {
+    res.status(401).send('Authentication required!');
+    return;
+  }
+  if (!isAuthorized(req.session.userId, req.params.formID, PERMISSIONS.ACTIONS.VOTE)) {
+    res.status(400).send('Unauthorized');
+    return;
+  }
+
+  // We must set the UserID to know who this ballot is associated to. This is
+  // only needed to allow users to cast multiple ballots, where only the last
+  // ballot is taken into account. To preserve anonymity, the web-backend could
+  // translate UserIDs to another random ID.
+  // bodyData.UserID = req.session.userId.toString();
+
+  // DEBUG: this is only for debugging and needs to be replaced before production
+  const bodyData = req.body;
+  console.warn('randomizing the SCIPER ID to allow for emulate votes - Use only for Development');
```

Comment:
Personal preference but I would have tried a more explicit warning,  I have a bad tendency to ignore non-caps warnings.


```suggestion
  console.warn(' DEV CODE - randomizing the SCIPER ID to allow for unlimited votes - Use only for Development');
```

Reply:
done 👍